wcivf/apps/parties/templates/parties/party_detail.html

```diff
@@ -57,6 +57,13 @@ <h2>{{ object.format_name }}</h2>
                 {% endif %}
             </div>
         {% endif %}
+
+        {% if object.register %}
+            {% blocktrans with register=object.format_register %}
```

Comment:
if you add `trimmed` to this block then the the translation file isn't going to contain the newliens and it will make things a little cleaner. Not a huge deal, but it does reduce the noise in diffs when we make changes in future.

Reply:
I've added `trimmed` now and fixed up :)